doc/advopt.txt

```diff
@@ -146,3 +146,5 @@ Advanced options:
                             see also https://nim-lang.github.io/Nim/estp.html
   --benchmarkVM:on|off      enable benchmarking of VM code with cpuTime()
   --sinkInference:on|off    en-/disable sink parameter inference (default: on)
+  --panics:on|off           turn runtime errors into process
```

Comment:
s/runtime error/defect/ ?
manual is clear about `defects` but ambiguous about `runtime error` (which can refer to both defect or catchable errors):

> Exceptions that indicate any other runtime error that can be caught inherit from system.CatchableError

Reply:
Good points but it remains "implementation specific" as the spec does not mandate compiler command line options.